src/any_agent/tools/a2a.py

```diff
@@ -68,7 +68,7 @@ async def a2a_tool_async(
     # with the traditional Optional[T] syntax for automatic function calling.
     # Using T | None syntax causes"Failed to parse the parameter ... for automatic function calling"
     async def _send_query(
-        query: str, task_id: Optional[str] = None, context_id: Optional[str] = None
+        query: str, task_id: str | None = None, context_id: str | None = None
```

Comment:
Why does my note on lines 66-68 no longer apply? If true, then please remove my note

Reply:
I messed something with the pre-commit somehow 🙏 Reverting these